react/game/src/app/screens/Game/components/Board/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import Square from './components/Square';
+import styles from './styles.scss';
+
+class Board extends Component {
+  renderSquare = i => <Square value={i} />;
+
+  render() {
+    const status = 'Next player: X';
+
+    return (
+      <React.Fragment>
```

Comment:
U can use a short syntax here `<>`
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Not supported in this version of Babel, instead i will be using Fragment stand alone and imported in top